### PR TITLE
Add Blinktrade (bitcambio.com.br) support

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Available APIs:
 * Kraken
 * Paymium
 * Poloniex
+* Blinktrade (Bitcambio)
```
Comment (OttoAllmendinger) [inline README.md]: please insert it alphabetically

Comment (jpereira) [reply README.md]: Ok, give me 1 minute.

Comment (jpereira) [reply README.md]: @OttoAllmendinger Done

File: src/ApiService.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ const {
   ProviderKraken,
   ProviderPaymium,
   ProviderPoloniex,
+  ProviderBlinktrade
```
Comment (OttoAllmendinger) [inline src/ApiService.js]: also alphabetical here

File: src/ApiService.js [hunk 3]
```diff
@@ -48,6 +49,7 @@ const Providers = {
   kraken: new ProviderKraken.Api(),
   paymium: new ProviderPaymium.Api(),
   poloniex: new ProviderPoloniex.Api(),
+  blinktrade: new ProviderBlinktrade.Api(),
```
Comment (OttoAllmendinger) [inline src/ApiService.js]: and here

File: src/ProviderBlinktrade.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const Lang = imports.lang;
+const Local = imports.misc.extensionUtils.getCurrentExtension();
+const BaseProvider = Local.imports.BaseProvider;
+
+const _invalidExchangeError = () =>
```
Comment (OttoAllmendinger) [inline src/ProviderBlinktrade.js]: unused, please remove

File: src/ProviderBlinktrade.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  apiDocs: [
+    ["API Docs", "https://blinktrade.com/docs/"]
+  ],
+
+  interval: 30, // unclear, should be safe
+
+  getUrl({ base, quote }) {
+    // log("DEBUG: Blinktrade: getUrl(): base="+base+", quote="+quote)
+
+    if (base !== undefined) {
```
Comment (OttoAllmendinger) [inline src/ProviderBlinktrade.js]: I think you meant `base === undefined`

please remove it though to not set surprising defaults, the user should just enter `BTC`/`BRL` himself

File: src/ProviderBlinktrade.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  apiName: "Blinktrade (Bitcambio)",
+
+  apiDocs: [
+    ["API Docs", "https://blinktrade.com/docs/"]
+  ],
+
+  interval: 30, // unclear, should be safe
+
+  getUrl({ base, quote }) {
+    // log("DEBUG: Blinktrade: getUrl(): base="+base+", quote="+quote)
```
Comment (OttoAllmendinger) [inline src/ProviderBlinktrade.js]: pls remove

File: src/ProviderBlinktrade.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    base = base.toUpperCase();
+    quote = quote.toUpperCase();
+
+    return `https://bitcambio_api.blinktrade.com/api/v1/${quote}/ticker?crypto_currency=${base}`;
+  },
+
+  getLast(data) {
+    // log("DEBUG: Blinktrade: result { vol_br: " + data.vol_brl+ ", vol: " + data.vol)
```
Comment (OttoAllmendinger) [inline src/ProviderBlinktrade.js]: pls remove

**Comment by jpereira** [comment]
@OttoAllmendinger done

**Comment by jpereira** [comment]
Plans to release it?

**Comment by OttoAllmendinger** [comment]
First I want to improve the handling of invalid ticker pairs a bit. The version I merged here still sets unexpected defaults, and the user has a hard time figuring out the accepted values.

**Pull request merged**